meerk40t/svgelements.py

```diff
@@ -190,22 +201,56 @@
     + "|"
     + SVG_TRANSFORM_SKEW_Y
 )
-PATTERN_TRANSFORM_UNITS = (
-    PATTERN_LENGTH_UNITS + "|" + PATTERN_ANGLE_UNITS + "|" + PATTERN_PERCENT
+PATTERN_TRANSFORM_UNITS = r"|".join(
+    (PATTERN_LENGTH_UNITS, PATTERN_ANGLE_UNITS, PATTERN_PERCENT)
 )
+TRANSFORM_UNIT_SINGULARS = {
+    "": "",
+    "%": "%",
+    "cm": "cm",
+    "cms": "cm",
+    "mm": "mm",
+    "mms": "mm",
+    "in": "in",
+    "ins": "in",
+    "inch": "in",
+    "inches": "in",
+    "mil": "mil",
+    "mils": "mil",
+    "pt": "pt",
+    "pts": "pt",
+    "deg": "deg",
+    "degs": "deg",
+    "grad": "grad",
+    "grads": "grad",
+    "gradian": "grad",
+    "gradians": "grad",
+    "rad": "rad",
+    "rads": "rad",
+    "radian": "rad",
+    "radians": "rad",
+    "turn": "turn",
+    "turns": "turn",
+}
```

Comment:
Many of these are not valid CSS length or angle units.

Reply:
Funnily enough I was thinking about what units of measure are defined in SVG when I was showering this morning. I was going to check.

So yes - you are right that most of this SVGelement change is incorrect, but there are few things that need to be retained - disallowing e.g. "turns" rather than "turn" for example, because "turns" is invalid SVG.